client/src/App.js

```diff
@@ -1,25 +1,31 @@
 import React, { Component } from 'react';
 import axios from 'axios';
 
-import FriendList from './FriendsList';
+import FriendList from './components/FriendsList';
+import FriendsForm from './components/FriendsForm';
 
 const API_URL = 'http://localhost:5000';
 class App extends Component {
   state = {
     friends: []
   };
 
-  componentDidMount() {
+  updateState = () => {
     axios
       .get(`${API_URL}/friends`)
```

Comment:
API_URL  makes things a lot more easier. Good work on getting your tasklist up for set up and making progress! When you work on the post again, think about how you would update each property in your data separately and not all at once on your post request. Great job! 👍 👍 👍